include/nigiri/location.h

```diff
@@ -26,9 +35,10 @@ struct location {
            timezone_idx_t,
            duration_t transfer_time,
            it_range<vector<location_idx_t>::const_iterator> equivalences);
+
   location_idx_t l_{location_idx_t::invalid()};
   std::string_view id_;
-  std::string_view name_;
+  cista::raw::generic_string name_;
```

Comment:
location is supposed to be a "view" on data, not own the data, so it should stay a std::string_view as it was before